src/body.rs

```diff
@@ -146,4 +146,43 @@ impl<T: 'static + Send + serde::Serialize> IntoResponse for Json<T> {
             .body(Body::from(serde_json::to_vec(&self.0).unwrap()))
             .unwrap()
     }
-}
\ No newline at end of file
+}
+
+impl <S: 'static> Extract<S> for String {
```

Comment:
@tzilist I think it would be nice to `rustfmt` the file here, it uses `impl<S: 'static> ...` rather than `impl <S: 'static>`.

Reply:
done! :)